RELEASE_NOTE.md

```diff
@@ -0,0 +1,76 @@
+# Release v0.13.0
+
+
+## New Features
+
+* PAI protocol:
+  - Introduce PAI protocol and job submission v2 ([#2260](https://github.com/microsoft/pai/pull/2260))
```

Comment:
Do you want to differ this "V2" with the previous "V2"?

Reply:
can you add the Protocol V2 introduction document (link) here?

Reply:
no, previous "v2" has been removed
link added